airflow/serialization/serde.py

```diff
@@ -147,6 +147,13 @@ def serialize(o: object, depth: int = 0) -> U | None:
         if is_serialized:
             return encode(classname, version, serialize(data, depth + 1))
 
+    # serialize namedtuple like tuples
+    if _is_namedtuple(o):
+        qn = "builtins.tuple"
+        data, _, version, is_serialized = _serializers[qn].serialize(o)
```

Comment:
This is redundant. Setting qn to "builtins.tuple" and move this check higher is sufficient.

Reply:
Indeed, refactored. Thanks !